src/apps/dashboard/features/logs/api/useServerLog.ts

```diff
@@ -0,0 +1,35 @@
+import { Api } from '@jellyfin/sdk';
+import { getSystemApi } from '@jellyfin/sdk/lib/utils/api/system-api';
+import { useQuery } from '@tanstack/react-query';
+import { useApi } from 'hooks/useApi';
+import type { AxiosRequestConfig } from 'axios';
+
+const fetchServerLog = async (
+    api?: Api,
+    name?: string,
```

Comment:
```suggestion
    api: Api,
    name: string,
```

I think we can just use a non-null assert on `api` and make `name` required.

Reply:
The useServerLog function uses `useApi()` to get the API instance, however that value can be undefined. I took this from the existing useServerLogs function.
The name parameter I can indeed make required.

Reply:
Since we are using `enabled: !!api` in `useQuery`, the query will never run unless `api` exists, and the check for it in `fetchServerLog` is pretty much only to make typescript happy.

So I think it's valid for a non-null assert like so:
```ts
queryFn: ({ signal }) => fetchServerLog(api!, name, { signal })
```